kubernetes/proto/nodetostatus/api.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto2";
```

Comment:
We shouldn't do this. Those protobuf definitions in k8s/proto are all generated ones.

Reply:
I don't see any necessity to define this proto in the first place.
NodeToStatusMap contains status, and status contains both reason and status code. But, in a real world, we don't need a reason in PostFilter. (at least, I don't come up with any usecase.)

So, let's simplify here -- use `nodename(string):statuscode(number)` map as nodeToStatusMap in wasm guest side so that we can remove this proto.

Reply:
Thanks. Fixed it. Used json.marshal and json.unmarshal